### ISSUE TITLE
Do we need to index the keyword URIs and (optionally) scheme name

### ISSUE THREAD

**Issue by dannymandel** [isamplesorg/isamples_inabox#306]
In the new core record format, the keywords has gone from a list of strings to a list of dictionaries, e.g.

```
        {
            "keyword": "Early Bronze Age",
            "keyword_uri": "http://vocab.getty.edu/page/aat/300019276",
            "scheme_name": "Styles and Periods Facet: Getty Art & Architecture Thesaurus"
        }
```

if we want to use the uri and scheme name, we'll need to change the solr schema.  If we want to keep it as a plain text string, we'll only need change the transformer to grab the `keyword` string out of the dictionary.  @datadavev please advise

**Comment by dannymandel** [comment]
The URIs are required.  We should solr facet on these.

**Comment by dannymandel** [comment]
See also the feature request from @ekansa: https://github.com/isamplesorg/isamples_inabox/issues/309 -- this work is required for that functionality.

**Comment by dannymandel** [comment]
@datadavev mentioned that this does bring up the question of what to do when there is no URI associated with a keyword, so we probably can't solely rely on these.

**Comment by smrgeoinfo** [comment]
I don't think we need to index the scheme name. If the users search with a URI, that implicitly gets the scheme. 
we do need to index all the keywords/keyword, to include keywords not from a vocabulary (scheme) or that have a URI.
